### ISSUE TITLE
show recently used mapping for same terms

### ISSUE THREAD

**Issue by AshkanA** [RinkeHoekstra/qber#69]
Show recently used selection (standardization) first – you want users to _consistently_ assign the same standardization for a given variable. 

If a user has the option to choose from 8 different variables called for example _year_, the next time this users wants to assign a standardization to _year_ it should be the same standardization. So providing the most recently used first would be more intuitive and consistent.

**Comment by RinkeHoekstra** [comment]
What do you mean by ‘the next time’, as one dataset will typically only contain one year variable. Do you mean across datasets?

**Comment by AshkanA** [comment]
yes and no..yes for other tables for example of the same dataset. And No like in cases where a term occurs multiple times in the columns. We had this with the census tables where you had (hierarchical) column with mannen - getrouwd and mannen - ongehuwd.
